configs/ASRock/B450-Gaming-ITX-ac.conf

```diff
@@ -0,0 +1,98 @@
+# ASRock B450 Gaming-ITX/ac
+# manufacturing date July 2019 (came with Zen2-aware BIOS)
+# board does have connectors for buzzer and 3 fans (2 case + 1 CPU)
+# board does not have a connector for intrusion detection
+# dmesg: "nct6775: Found NCT6792D or compatible chip at 0x2e:0x290"
+
+chip "nct6792-*"
+
+   # beeping appears to not work
+   set beep_enable 1
+
+   # voltages
+
+   label   in0     "Vcore"
+   compute in0     @*2, @/2
+   set     in0_min 0.16
+   set     in0_max 1.46
+   set     in0_beep 1
+
+   label   in2     "AVCC"
+   set     in2_min 3.30 * 0.90
+   set     in2_max 3.30 * 1.10
+
+   label   in3     "+3.3V"
+   set     in3_min 3.30 * 0.90
+   set     in3_max 3.30 * 1.10
+
+   label   in7     "3VSB"
+   set     in7_min 3.30 * 0.90
+   set     in7_max 3.30 * 1.10
+
+   label   in8 "VBAT"
+   set     in8_min 3.00 * 0.90
+   set     in8_max 3.30 * 1.10
+
+   label   in9 "+12V"
+   compute in9 @*(53/8), @/(53/8)
+   set     in9_min 12 * 0.90
+   set     in9_max 12 * 1.10
+
+   label   in13     "+5V"
+   compute in13 @*(24/8), @/(24/8)
+   set     in13_min 5 * 0.90
+   set     in13_max 5 * 1.10
+
+   # these are all zero
+   ignore  in1
+   ignore  in4
+   ignore  in5
+   ignore  in6
+   ignore  in14
+
+   # these have non-zero input, but are unknown
+   ignore  in10
+   ignore  in11
+   ignore  in12
+
+   # temperatures
+   # not sure about temp2 and temp3 
+   label   temp2 "VRM(?) temp"
+   set temp2_max      55
+   set temp2_max_hyst 50
+   set temp2_beep 1
+   label   temp3 "MB(?) temp"
+   set temp3_max      55
+   set temp3_max_hyst 50
+   set temp3_beep 1
+   label   temp7 "CPU temp"
+
+   # temp2 has similar readings to temp3
+   # other temps are zero or unreasonably high/low
+   ignore  temp1
+   ignore  temp4
+   ignore  temp5
+   ignore  temp6
+   ignore  temp8
+   ignore  temp9
+   ignore  temp10
+
+   # fans
+
+   label fan1 "SYS fan"
+   set fan1_min 300
+   set fan1_beep 1
+   label fan2 "CPU fan"
+   set fan2_min 600
+   set fan2_beep 1
+
+   ignore  fan0
```

Comment:
The board seems to have 3 fan headers, whereas your config has only two that are not `ignore`d. What's up with the third one?

Reply:
resolved this by connecting a spare fan. config now also reflects labels on board.